docs/dev/tools/cli.md

```diff
@@ -279,6 +279,187 @@ viam organizations --help
+| Argument | Description |
+| -------- | ----------- |
+| `filter` | `add` or `remove` images or tags from a dataset using a filter. See [Using the `filter` argument](#using-the-filter-argument).|
+| `ids` | `add` or `remove` images or tags from a dataset by specifying one or more binary data IDs as a comma-separated list. See [Using the `ids` argument](#using-the-ids-argument).|
+| `--help` | Return help |
+
+##### Named arguments
+
+<!-- prettier-ignore -->
+| Argument | Description | Applicable commands | Required? |
+| -------- | ----------- | ------------------- | --------- |
+| `--destination` | Output directory for downloaded data. | `export tabular`, `export binary` | **Required** |
+| `--component-name` | Filter by specified component name. | `export binary`, `delete`, `tag filter`| Optional |
+| `--component-type` | Filter by specified component type. | `export binary`, `delete`, `tag filter` | Optional |
+| `--component-model` | Filter by specified component model. | `export`, `delete`| Optional |
+| `--delete-older-than-days` | Number of days, 0 means all data will be deleted. | `delete` | Optional |
+| `--timeout` | Number of seconds to wait for file downloads. Default: `30`. | `export binary` | Optional|
+| `--start` | ISO-8601 timestamp indicating the start of the interval. | `export binary`, `export tabular`, `delete`, `dataset`, `tag filter`| Optional |
+| `--end` | ISO-8601 timestamp indicating the end of the interval. | `export binary`, `export tabular`, `delete`, `dataset`, `tag filter`| Optional |
+| `--binary-data-ids` | Binary data IDs to add or remove tags from. | `tag ids` | **Required** |
+| `--location-ids` | Filter by specified location ID (accepts comma-separated list). See [Using the `ids` argument](#using-the-ids-argument) for instructions on retrieving these values. | `export binary`, `delete`, `tag filter`| Optional |
+| `--method` | Filter by specified method. | `export binary`, `export tabular`, `delete`, `tag filter`| Optional |
+| `--mime-types` | Filter by specified MIME type (accepts comma-separated list). | `export binary`, `delete`, `tag filter`|false |
+| `--org-ids` | Filter by specified organizations ID (accepts comma-separated list). See [Using the `ids` argument](#using-the-ids-argument) for instructions on retrieving these values. | `export binary`, `delete`, `tag filter`| Optional |
+| `--parallel` | Number of download requests to make in parallel, with a default value of 10. | `export binary`, `delete`, `dataset export` | Optional |
+| `--part-id` | Filter by specified part ID. | `export binary`, `export tabular`, `delete`, `tag filter`| Optional, **Required** for `export tabular` |
+| `--part-name` | Filter by specified part name. | `export binary`, `delete`, `tag filter`| Optional |
+| `--machine-id` | Filter by specified machine ID. | `export binary`, `delete`, `tag filter` | Optional |
+| `--machine-name` | Filter by specified machine name. | `export binary`, `delete`, `tag filter`| Optional |
```

Comment:
Flagging this since it came up today: many commands have a `--machine` option, aliased to `--machine-name` and `--machine-id`, which you can pass either a machine id or machine name to. Curious that this list includes the aliases but not the parent option (when our other docs explicitly mention only the parent option, with one notable exception where there is no alias).